templates/institutional/index.json-ld

```diff
@@ -29,6 +29,16 @@
 				"language": "fr",
 				"path": "institution-fr.html"
 			},
+			{
+				"title": "[Institution Name] - landing page (provisional)",
+				"language": "en",
+				"path": "institution-landing-en.html"
+			},
+			{
+				"title": "[Nom de l’institution] - Pages d'accueil (provisoire)",
+				"language": "fr",
+				"path": "institution-landing-fr.html"
+			},
```

Comment:
Add a property inside the page description object that provide the sub-status of those template, like `"status": "provisional"`